### PR TITLE
SBT-64 Add Logging for BioSequence, Sample, SamplesContainer + docs

### PR DESCRIPTION
**Pull request by quantori-cherednichenko**

### CODE REVIEW

File: src/signals_notebook/entities/biological_sequence.py [hunk 1]
```diff
@@ -31,6 +31,20 @@ def create(
...
+
+        Args:
+            container: Container where create new BiologicalSequence
+            name: file name
+            content: BiologicalSequence content
+            file_extension: BiologicalSequence extension
+            force: Force to post attachment
+
+        Returns:
+            Image
```
Comment (quantori-cherednichenko) [inline src/signals_notebook/entities/biological_sequence.py:44]: it will be fixed

Comment (quantori-cherednichenko) [inline src/signals_notebook/entities/biological_sequence.py:44]:
```suggestion
           BiologicalSequence
```

File: src/signals_notebook/entities/biological_sequence.py [hunk 2]
```diff
@@ -31,6 +31,20 @@ def create(
...
+
+        Args:
+            container: Container where create new BiologicalSequence
+            name: file name
+            content: BiologicalSequence content
+            file_extension: BiologicalSequence extension
+            force: Force to post attachment
+
+        Returns:
+           BiologicalSequence
```
Comment (quantori-cherednichenko) [inline src/signals_notebook/entities/biological_sequence.py:44]:
```suggestion
            BiologicalSequence
```

**Review by quantori-shishkin — DISMISSED** [review]

File: src/signals_notebook/entities/samples/sample.py [hunk 3]
```diff
@@ -65,6 +68,7 @@ def __getitem__(self, index: Union[int, str, UUID]) -> SampleCell:
         if isinstance(index, UUID):
             return self._cells_by_id[index]
 
+        log.exception('IndexError were caught. Invalid index')
```
Comment (quantori-pokidovea) [inline src/signals_notebook/entities/samples/sample.py:71]: I don't see, why do we need this

File: src/signals_notebook/entities/samples/samples_container.py [hunk 4]
```diff
@@ -35,6 +38,7 @@ def __getitem__(self, index: Union[int, str, UUID, EID]) -> Sample:
         if isinstance(index, EID):
             return self._samples_by_id[index]
 
+        log.exception('IndexError were caught. Invalid index')
```
Comment (quantori-pokidovea) [inline src/signals_notebook/entities/samples/samples_container.py:41]: The same

**Review by quantori-pokidovea — APPROVED** [review]

**Pull request merged**